enterprise_access/apps/api/serializers.py

```diff
@@ -141,7 +148,69 @@ def validate_content_key(self, value):
 
         return value
 
-class SubsidiyAccessPolicyListSerializer(SubsidiyAccessPolicyRedeemSerializer):  # pylint: disable=abstract-method
+
+class SubsidyAccessPolicyCRUDSerializer(serializers.ModelSerializer):
+    """
+    Serializer to validate policy data for CRUD operations.
+    """
+    uuid = serializers.UUIDField(read_only=True)
+    policy_type = serializers.ChoiceField(choices=PolicyTypes.CHOICES)
+    description = serializers.CharField(max_length=None, min_length=None, allow_blank=False, trim_whitespace=True)
+    active = serializers.BooleanField()
+    group_uuid = serializers.UUIDField(allow_null=False)
+    catalog_uuid = serializers.UUIDField(allow_null=False)
+    subsidy_uuid = serializers.UUIDField(allow_null=False)
+    access_method = serializers.ChoiceField(choices=AccessMethods.CHOICES)
+
+    per_learner_enrollment_limit = serializers.IntegerField()
+    per_learner_spend_limit = serializers.IntegerField()
+    spend_limit = serializers.IntegerField()
```

Comment:
The above 3 fields are not applicable to every policy. So should we check/validate or specify that which field will be used for which policy?

Reply:
Or maybe we could just discard the other two limits if they are non-zero in serializer validate method?